staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -2262,7 +2347,7 @@ func TestCELValidationContextCancellation(t *testing.T) {
 
 // This is the most recursive operations we expect to be able to include in an expression.
 // This number could get larger with more improvements in the grammar or ANTLR stack, but should *never* decrease or previously valid expressions could be treated as invalid.
-const maxValidDepth = 243
+const maxValidDepth = 250
```

Comment:
Is this consider a breaking change? Having a 1.28 expression and then rollback to earlier version might fail the validation?